packages/sitecore-jss-react/package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint ./src/**/*.tsx ./src/**/*.ts",
     "test": "mocha --require ts-node/register --require ./src/tests/shim.ts ./src/tests/jsdom-setup.ts ./src/tests/enzyme-setup.ts \"./src/**/*.test.ts\" \"./src/**/*.test.tsx\"",
     "prepublishOnly": "npm run build",
-    "coverage": "nyc --require ts-node/register npm test"
```

Comment:
Removing this means the coverage only includes things that have unit tests. So things like `VisitorIdentification.tsx`, which have 0 tests will not show up in the report. 

I remember fighting with this on the nextjs package ... Looks like in this case, you might just have to add the following to the top of the `utils.ts` to fix the 'style-attr' type definitions error:
```
// eslint-disable-next-line spaced-comment
/// <reference types="../global" />
```

Reply:
@ambrauer - got it. This package didn't even have nyc installed so we were not running a coverage report for it. After adding 'nyc' I made the script match what I saw us using in other packages. Specifically, the nextjs package also uses `"coverage": "nyc npm test"`. Seems like that one needs to be updated as well, or is there a reason why that one is different?

Reply:
As discussed with @ambrauer, will create a separate ticket for looking into the coverage script of react and nextjs packages